sklearn/utils/estimator_checks.py

```diff
@@ -347,7 +349,9 @@ def _construct_instance(Estimator):
     required_parameters = getattr(Estimator, "_required_parameters", [])
     if len(required_parameters):
         if required_parameters in (["estimator"], ["base_estimator"]):
-            if issubclass(Estimator, RegressorMixin):
+            if issubclass(Estimator, RANSACRegressor):
```

Comment:
```suggestion
            if issubclass(Estimator, RANSACRegressor):
                # `RANSACRegressor` will raise an error with any model other
                # than `LinearRegression` if we don't fix `min_samples` parameter.
                # For common test, we can enforce using `LinearRegression` that
                # is the default estimator in `RANSACRegressor` instead of `Ridge`.
```